browser/src/slideshow/engine/SlideShowNavigator.ts

```diff
@@ -129,6 +129,24 @@ class SlideShowNavigator {
 		this.displaySlide(this.theMetaPres.numberOfSlides - 1, true);
 	}
 
+	goToSlideAtBookmark(bookmark: string) {
+		NAVDBG.print(
+			'SlideShowNavigator.goToSlideAtBookmark: ' +
+				bookmark +
+				' current index: ' +
+				this.currentSlide,
+		);
+		for (let i = 0; i < this.theMetaPres.numberOfSlides; i++) {
```

Comment:
I wonder if it could make more sense to iterate on core and set the `bookmark` parameter to the slide hash value instead of the slide name, so we save the need to retrieve and forward the slide name.
iterate on core or iterate on online ?  @eszkadev, @quikee ?